test-native/bb.edn

```diff
@@ -0,0 +1,24 @@
+{:tasks
+ {:requires ([babashka.fs :as fs])
+  :init (do
+          (def jar "target/native-test-0.0.1-standalone.jar")
+          (def native (if (fs/windows?)
+                        "native-test-0.0.1-standalone.exe"
+                        "./native-test-0.0.1-standalone")))
+  prep {:doc "Prepares httpkit for usage in clj"
+        :task (when-not (fs/exists? "../../target/classes")
+                (clojure "-X:deps prep"))}
+  uber {:doc "Makes uberjar"
+        :depends [prep]
+        :task (when (seq (fs/modified-since jar ["src"]))
+                (clojure "-T:build uber"))}
+  native:build {:doc "Compile native image"
+                :depends [uber]
+                :task (when (seq (fs/modified-since native jar))
+                        (shell "native-image"
+                               "--initialize-at-build-time=clojure,org.httpkit,native"
+                               "--no-fallback"
+                               "-jar" jar))}
+  native:test {:doc "Run native test"
```

Comment:
This kicks off the whole chain of build steps necessary to build the native image and then runs it.